app/src/main/java/org/wikipedia/analytics/eventplatform/AppSessionEvent.kt

```diff
@@ -90,7 +90,7 @@ class AppSessionEvent {
 
     @Suppress("unused")
     @Serializable
-    @SerialName("/analytics/mobile_apps/android_app_session/1.0.0")
+    @SerialName("/analytics/mobile_apps/app_session/1.0.0")
     class AppSessionEventImpl(@SerialName("length_ms") private val length: Int,
                               @SerialName("session_data") private val sessionData: SessionData,
                               private val languages: List<String>) :
```

Comment:
Looks good, tested and saw events coming in.. Just a minor change.. 
Could you please change the session id to the new app_session instead of the old stream android.app_session.